CONTRIBUTING.md

```diff
@@ -13,39 +13,26 @@ On a command line build the project with Maven to run the unit tests and integra
 
 ## Integration Tests for Notify dev team
 
-You will need to source an environment.sh file. The contents of that file are explained below. All secrets are found in the jenkins vault, stored in ansible in the notifications-aws repo.
+You will need to source an environment.sh file. You can find a copy in notify-pass credentials/client-integration-tests. You will also need to include the following variable.
 
-```
+```shell
 export NOTIFY_API_URL=https://api.notify.works
-export API_KEY= # vault_jenkins_notify_client.api_key
-export FUNCTIONAL_TEST_EMAIL=notify-tests-preview+client_funct_tests@digital.cabinet-office.gov.uk
-export FUNCTIONAL_TEST_NUMBER=+447537417379 # Twilio number
-export EMAIL_TEMPLATE_ID=f0bb62f7-5ddb-4bf8-aac7-c1e6aefd1524
-export SMS_TEMPLATE_ID=31046c06-418a-49bf-86de-706b68415b47
-export LETTER_TEMPLATE_ID=de1252c4-d8c3-4435-92fb-02f136778b2b
-export EMAIL_REPLY_TO_ID=db8d1a9d-41ef-43cd-a04a-ed7d95214d95
-export SMS_SENDER_ID=e9355456-c52f-4648-abb6-5fc192b29195
-export INBOUND_SMS_QUERY_KEY= # vault_jenkins_notify_client.inbound_sms_query_key
-export API_SENDING_KEY= # vault_jenkins_notify_client.api_sending_key
 ```
 
-
-## Update version
-Increment the version in the `src/main/resources/application.properties` and `pom.xml` files.
-
-
-## Deploying
-
-[For internal notify use only]
-You'll need to make sure you have the `notify-gpg-key` private key available locally.
+Use the following command to run the integration tests:
+```shell
+mvn clean verify
+```
+NOTE: you'll get a build failure `[ERROR] Failed to execute goal org.apache.maven.plugins:maven-gpg-plugin:1.5:sign (sign-artifacts) on project notifications-java-client: Exit code: 2 -> [Help 1]
+` unless you import the gpg key, however, the you should see if the integration tests failed in the log message before the error. Import the key with this command:
```

Comment:
there shouldn't be a need to run this manually any more, can we remove this comment about how to import the GPG key (and the code snippet below). if people need to for whatever reason, we can always reverse-engineer the steps from the concourse job

Reply:
The verify phase signs the artifact, which needs the gpg key. The test will pass without it but you still get a build failure

Reply:
i think ideally we figure out the pom.xml and add a separate build keyword that runs the integration tests plugin without running the gpg signing plugin. 

More importantly than seeing an exit code of 0 is that I don't really want any of us to have the concourse key on our local keychains if we can help it